src/ui/components/Queue.tsx

```diff
@@ -255,7 +277,22 @@ const QueueActions = (props: {
   )
 }
 
-// FIXME: typings
+interface QueueProps {
+  queue: AppQueue
+  selectedStatus: Status
+  selectStatus: (statuses: Record<string, Status>) => void
+  cleanAllDelayed: () => Promise<void>
+  cleanAllFailed: () => Promise<void>
+  retryAll: () => Promise<void>
+  retryJob: (job: AppJob) => () => Promise<void>
+}
+
+// REVIEW: this is in place so we can manipulate the return type
+//  of Object.keys which is string[] by default
+// We need to extend so babel doesn't think it's JSX
```

Comment:
Looks good. Could keep either the entire comment (except for `REVIEW`) or just line 292 🙂

Reply:
Yea, 292 is supposed to be a "permanent" comment here, hence the lack of indentation :p~